.env.example

```
@@ -0,0 +1,42 @@
+DB_USER=root
+DB_PASSWORD=password
+MYSQL_ROOT_PASSWORD=password
+MYSQL_DATABASE=rspec_boilerplate_development
+
+# Rails Configuration
+RAILS_ENV=development
+DATABASE_URL=mysql2://root:password@mysql:3306/rspec_boilerplate_development
+
+# PostgreSQL Configuration (Legacy/DevContainer)
+POSTGRES_PASSWORD=secret
+POSTGRES_DATABASE=rspec_boilerplate_legacy_development
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The POSTGRES_DATABASE key should go before the POSTGRES_PASSWORD key

(UnorderedKey)